### DISCUSSION TITLE
New to SharePoint

### DISCUSSION THREAD

**Discussion by mouzzampk** [pnp/powershell#4459] [General]
Hiya, I know this might be a basic question but I'm still learning. 
Can you help me with the tenant ID or host name, please?

I'm registering a new app with full control and my site is https://mbmouzzampk.com/sites/Finance

I'm not sure how to find the -Tenant. 

Thanks!

**Comment by Barbarur** [comment]
-Tenant is actually your AzureAD domain; mbmouzzampk.onmicrosoft.con or mbmouzzampk.com

**Reply by mouzzampk** [reply]
Thank you so much 😇
